app-misc/elasticsearch/elasticsearch-8.2.0.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd tmpfiles
+
+DESCRIPTION="Free and Open, Distributed, RESTful Search Engine"
+HOMEPAGE="https://www.elastic.co/elasticsearch/"
+SRC_URI="https://artifacts.elastic.co/downloads/${PN}/${P}-linux-x86_64.tar.gz"
+LICENSE="Apache-2.0 BSD-2 Elastic-2.0 LGPL-3 MIT public-domain"
+SLOT="0"
+KEYWORDS="~amd64"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-env.patch"
+)
+
+BDEPEND="acct-group/elasticsearch
+	acct-user/elasticsearch"
```

Comment:
I think `DEPEND` is more appropriate for these.

Reply:
@Flowdalic suggested that they should be `BDEPENDS` as they're used in `src_install`.

I'm easy either way, but I can address this for the next version bump if there's a reason to do so.

Reply:
5f3e5a6c9fc96ea431bb2ecf9a62bcd59e81f6a8 I updated it while merging this. I don't really know whether it's right or not, since you can't change ownership to a non-existing user:group, but then again does it check DEPEND or BDEPEND while calling `chown` :) 

I'd say until someone clarifies this DEPEND is the more secure choice.